app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -724,8 +893,23 @@ private void swipeLeftOnHabit(Habit habitToDelete) {
         _solo.drag(fromX, location[0], fromY, fromY, 10);
     }
 
+    private void swipeLeftOnHabitEvent(HabitEvent eventToDelete) {
+        TextView view = _solo.getText(eventToDelete.getComment());
 
-    private Habit goToViewHabit() {
+        // locate row
+        int[] location = new int[2];
+        view.getLocationOnScreen(location);
+
+        int displayWidth = _solo.getCurrentActivity().getWindowManager().getDefaultDisplay().getWidth();
+
+        int fromX = displayWidth - 10;
+        int fromY = location[1];
+        _solo.drag(location[0], 0, fromY, fromY, 10);
+        _solo.drag(location[0], 0, fromY, fromY, 10);
```

Comment:
Minor: This may need to be updated after the double swipe PR goes in! Don't worry about it here though. I guess it's a race to whoever merges in first 😂

Reply:
Yep I was just thinking about that too haha. I'll submit an issue to fix it otherwise